feature/system/gnmi/cliorigin/mixed_oc_cli_origin_support_test/mixed_oc_cli_origin_support_test.go

```diff
@@ -46,9 +46,10 @@ func buildOCUpdate(path *gpb.Path, value string) *gpb.Update {
 	if len(path.GetElem()) == 0 || path.GetElem()[0].GetName() != "meta" {
 		path.Origin = "openconfig"
 	}
+	json_value := fmt.Sprintf("\"%s\"", value)
 	update := &gpb.Update{
 		Path: path,
-		Val:  &gpb.TypedValue{Value: &gpb.TypedValue_StringVal{StringVal: value}},
+		Val:  &gpb.TypedValue{Value: &gpb.TypedValue_JsonIetfVal{JsonIetfVal: []byte(json_value)}},
```

Comment:
This change might probably break our current tests. We usually run into this error: "error gathering JSON paths for coverage:json: cannot unmarshal string into Go value of type map[string]interface {}". Why cant we keep it as a StringVal itself? Why is this change necessary?

Reply:
Neither 'json' nor 'string' have a canonical way to represent yang data. JSON_IETF is specified as RFC7951, and is the only format supported for yang data in gnmi for at least IOS XR. If 'cli' data is being set, 'string' can be used.